examples/pygstc/simple_pipeline.py

```diff
@@ -0,0 +1,167 @@
+    if (self.pipe_exists("p0")):
+      self.gstc.pipeline_delete("p0")
+
+    self.gstc.pipeline_create("p0", self.pipeline)
+    self.pipe_created = True
+    self.gstc.pipeline_play("p0")
+
+  def pauseVideo(self):
+    print("Playing")
+    self.gstc.pipeline_pause("p0")
+
+  def continueVideo(self):
+    print("Playing")
+    self.gstc.pipeline_play("p0")
+
+  def stopVideo(self):
+    print("Stop")
+    if (self.pipe_exists("p0")):
+      self.gstc.pipeline_stop("p0")
+      self.gstc.pipeline_delete("p0")
+      self.pipe_created = False
+
+  def setRes(self, width, height):
+    print("Changing video resolution")
+    if (self.pipe_exists("p0")):
+      self.gstc.pipeline_delete("p0")
+    self.gstc.pipeline_create("p0", self.pipeline)
+    self.gstc.element_set("p0", "cf", "caps", "video/x-raw,width="+width+",height="+height+"")
+    self.gstc.pipeline_play("p0")
+
+  def setSpeed(self, speed):
+      self.gstc.event_seek("p0", rate=speed, format=3, flags=1, start_type=1, start=0, end_type=1, end=-1)
+
+  def jumpTo(self, position):
+    self.gstc.event_seek("p0", rate=1.0, format=3, flags=1, start_type=int(position*10^9), start=0, end_type=0, end=0)
+
+  def pipe_exists(self, pipe_name):
+    #Check if pipe is already created
+    existing_pipes = self.gstc.list_pipelines()
+    if (existing_pipes == []):
+      ret = False
+    elif( existing_pipes[0]['name'] == pipe_name):
+      ret = True
+    else:
+      ret = False
+    return ret
+
+  def printUsage(self):
+    print("play: To play and run")
+    print("pause: To pause the video")
+    print("continue: To continue after paused")
+    print("stop: To stop and close the playing")
+    print("set_res $WIDTH $HEIGHT: To change the video resolution")
+
+def printError():
+  print("To use run: python3 simple_pipeline.py $VIDEO_PATH")
+
+
+if __name__ == "__main__":
+  print("Sample PyGstC Video Player")
+
+  if(len(sys.argv) != 2):
+    printError()
+    sys.exit(0)
+
+  try:
+    myPlayer = GstcPlayer()
+    myPlayer.openFile(sys.argv[1])
+    myPlayer.playVideo()
+    action = ["running"]
+
+    while (action[0] != "exit"):
+
+      if (action[0]=="play"):
+        myPlayer.playVideo()
+      elif (action[0]=="pause"):
+        myPlayer.pauseVideo()
+      elif (action[0]=="continue"):
+        myPlayer.continueVideo()
+      elif (action[0]=="stop"):
+        myPlayer.stopVideo()
+      elif (action[0]=="set_speed"):
+        if (len(action) == 2 and abs(float(action[1])) <= 1):
+          myPlayer.setSpeed(float(action[1]))
+        else:
+          print("Playback speed valid range: [-1, 1]. 0 is not allowed")
+
+      elif (action[0] == "play_fw"):
+        myPlayer.setSpeed(float(1))
+
+      elif (action[0] == "play_bw"):
+        myPlayer.setSpeed(float(-1))
+
+      elif (action[0]=="jump"):
+        if (len(action) == 2 and action[1].isnumeric() and int(action[1]) >= 0):
+          myPlayer.jumpTo(int(action[1]))
+        else:
+          print("jump $NANO_SECS")
```

Comment:
these are not nano seconds. You receive seconds (you convert them manually later).